README.md

```diff
@@ -50,42 +55,49 @@ Install the packages from `requirements.txt`
 
 ## Training an agent
 
-To train an agent for 1000 timesteps on the vibrating bridge environment and save it as `my_first_agent.pkl`, navigate to the `complex_envs` folder and run the following command:
+To train an agent for 40,000 timesteps on the vibrating bridge environment and save it as `trained_agents/my_first_agent.pkl`, run the following command:
 
 ```
-$ cd complex_envs
-$ python train.py -n 1000 -m my_first_agent
+$ python src/train.py -n 40000 -m trained_agents/my_first_agent
 ```
-This command will also produce a TensorBoard folder at `./tensorboard_log` that can be visualized with
+This command will also produce a TensorBoard folder at `/tensorboard_log` that can be visualized by running the following command from the same directory
 
 ```
 $ tensorboard --logdir tensorboard_log/
 ```
 ## Rolling out a trained agent
 
-To rollout a trained agent that is stored at `complex_envs/my_first_agent.pkl` for 60 steps, run the following command from the same folder:
+To rollout a trained agent that is stored at `trained_agents/my_first_agent.pkl` for 60 steps, run the following command:
 
 ```
-$ python rollout.py -n 60 -i my_first_agent.pkl
+$ python src/rollout.py -n 60 -i trained_agents/my_first_agent.pkl -f rollouts/output.npz
 ```
 
-The rollout will be saved as `output.npz`, which can be changed by passing `-f <filename>` to the above command.  Note that the trajectories produced will have length equal to the number of rollout steps + the number of warmup steps + the number of equilibriation steps (values of which are set in `/configs/config.yml`).
+The rollout will be saved as `rollouts/output.npz`, which can be changed by passing `-f <filename>` to the above command.  Note that the trajectories produced will have length equal to the number of rollout steps + the number of warmup steps + the number of equilibriation steps (values of which are set in `/configs/config.yml`).
 
 ### Visualizing a rollout
 
-To visualize a rollout saved in `complex_envs/output.npz`, run the following command from the same folder:
+To visualize a rollout saved in `rollouts/output.npz`, run the following command:
 
 ```
-# python visualize.py -i output.npz -f first_output
+# python src/visualize.py -i rollouts/output.npz -f rollouts/first_output
 ```
```

Comment:
Toward the end of the readme, it would be best to include a note about which variables users should experiment with if they were to run your repo: highlight 1 or 2, describe what they do, and what the bounds are for changing them